docs/1-Using-Fleet/configuration-files/vulnerability_processing.yml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: v1
+kind: config
+spec:
+  vulnerability_settings:
```

Comment:
Thank you for updating the config files! 

I suggest that, instead of adding a new `vulnerability_processing.yml` file, we add the new `vulnerability_settings` key into the existing [`configuration-files/multi-file-configuration/organization-settings.yml`](https://github.com/fleetdm/fleet/blob/main/docs/1-Using-Fleet/configuration-files/multi-file-configuration/organization-settings.yml) file. This way, users that look at the example `organization-settings.yml` file, they'll see `vulnerability_processing` alongside all the other available options.